tests/e2e/monitoring_test.go

```diff
@@ -222,3 +233,226 @@ func (tc *MonitoringTestCtx) ValidateMonitoringCRDeleted(t *testing.T) {
 	// Ensure Monitoring CR is removed because of ownerreference
 	tc.EnsureResourcesGone(WithMinimalObject(gvk.Monitoring, types.NamespacedName{Name: "default-monitoring"}))
 }
+
+func getTempoMonolithicName(dsci *dsciv1.DSCInitialization) string {
+	return "data-science-tempomonolithic"
+}
+
+func getTempoStackName(dsci *dsciv1.DSCInitialization) string {
+	return "data-science-tempostack"
+}
```

Comment:
guess we do not need function here any more?

Reply:
they're still being used in multiple places in the test case, it'd probably be okay to leave them in case the naming changes in the future